include/ibamr/ConstraintIBMethod.h

```diff
@@ -333,6 +375,16 @@ class ConstraintIBMethod : public IBAMR::IBMethod
      * \brief Calculate Eulerian Momentum.
      */
     void calculateEulerianMomentum();
+    
```

Comment:
For consistency: calculateEulerianMomentum() and calculateLagrangianMomenum() or calculateStructureMomentum()?

(Consistent naming conventions can make it easier to use the code. We should probably adopt some...)